### PR TITLE
fixed requirements for google colab

### PR DESCRIPTION
**Pull request by orperel**
Little updates to requirements to be able to run nglod training (`app.main`) on Google Colab:

1. pysdf missing from requirements --> added to requirements.txt
2. Some flows use `torch.clip`, which is only available from torch 1.7.0 and onwards (tested with CUDA 11) --> requirements.txt updated
3.  MeshDataset has a stale `import spc` statement which fails the training code --> removed from MeshDataset

Another change required to run training, and not included in this pull-request:
The `armadillo_normalized.obj` model is n/a, so possibly the normalization code of MeshDataset should be uncommented (I'm not sure how other flows may be affected).

### CODE REVIEW

**Comment by tovacinni** [comment]
Thanks for this contrib! A few points:

1. The `pysdf` dependency isn't actually needed (I've deleted it off the main branch just now) so the requirements don't need to be updated
2. I think `torch.clip` is an alias for `torch.clamp` which was introduced in 1.7.0. I'll replace all the clips with clamps and I think that should fix the versioning problem. 

If you can update these 2 things I'll merge this branch, thanks!

**Comment by tovacinni** [comment]
Also thanks for pointing out the normalization issue- this was an update in the recent refactor. It has now been uncommented and the README is updated again to make this more clear.

**Comment by orperel** [comment]
Done
